src/core/utils.jl

```diff
@@ -378,3 +402,16 @@ function convert_gal_to_kwh(delta_T_degF::Real, rho_kg_per_m3::Real, cp_kj_per_k
     kwh_per_gal = kj_per_gal / 3600.0  # divide by kJ/kWh, i.e., sec/hr, to get: [kWh/gal]
     return kwh_per_gal
 end
+
+"""
+    The input offgrid_other_capital_costs is considered to be for depreciable assets. 
+    Straight line depreciation is applied, and the depreciation expense is assumed to reduce the owner's taxable income
+    TODO: should the savings array start at [0] or [1] ??
```

Comment:
I don't understand the question for this TODO. Can we address it?

Reply:
depreciation starts at the end of the first year  (assuming that all capital costs are paid at the beginning of the first year).

Reply:
That looks correct when comparing to the MACRS depreciation schedule https://github.com/NREL/REopt.jl/blob/ce7b79ee9804ba12e8d43ba664f36fee8a24f333/src/core/utils.jl#L129

Does the test value now not match the API value? Is that because the API assumes depreciation at the beginning of year 1?

Reply:
Yes, agreed regarding alignment with tax_savings_array. We do not currently apply depreciation savings to the `offgrid_other_capital_costs` in the API. This is a new change that was suggested by Dan and Doug.